### PR TITLE
(fix) Fix e2e test for highlight

### PR DESCRIPTION
**Pull request by mdhaman**
@ioanpocol , @petrjasek , @mugurrus  fix for the highlight test.

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
just wondering - would it be possible to handle undefined param in that filter? not to complicate the template users might need to deal with?

**Comment by akintolga** [comment]
Yes, I think it is possible, I'll make a change for that today.

**Comment by mdhaman** [comment]
@petrjasek done the change.

**Comment by petrjasek** [comment]
:+1: nice

File: features/highlights.feature [hunk 1]
```diff
@@ -336,3 +336,70 @@ Feature: Highlights
...
+        When we get "/archive"
+        Then we get list with 2 items
+
+        When we post to "generate_highlights"
+        """
+        {"package": "package", "preview": true}
+        """
+        Then we get new resource
+        """
+        {"type": "text"}
```
Comment (ahilles107) [inline features/highlights.feature]: missing new line at end of file

**Pull request merged**